src/c-st-ext.adoc

```diff
@@ -548,14 +544,14 @@ These instructions use the CR format.
 
 C.JR (jump register) performs an unconditional control transfer to the
 address in register _rs1/cs1_. C.JR expands to `jalr x0, 0(rs1)`, or
-`jalr {creg}0, 0(cs1)` for {cheri_base_ext_name}.
+`jalr {creg}0, 0({creg_pref}s1)` for {cheri_base_ext_name}.
```

Comment:
Aren't these the same now? Both expand to `jalr x0, 0(rs1)`

Reply:
yep - it _is_ a lot simpler - will fix